src/core/qgscoordinatereferencesystem.h

```diff
@@ -45,9 +45,13 @@ class QgsCoordinateReferenceSystemPrivate;
 struct PJconsts;
 typedef struct PJconsts PJ;
 
+#if PROJ_VERSION_MAJOR>7 || (PROJ_VERSION_MAJOR==7 && PROJ_VERSION_MINOR>2)
```

Comment:
This change will only be introduced in PROJ 8, so I don't think you need the second part

Reply:
is master at 8?